frontend/src/components/AppBar.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import { Link } from 'react-router-dom';
+
+import './AppBar.css';
+
+const pages = ['Documents', 'Tags'];
+
+function responsiveAppBar() {
+  return (
+    <AppBar position="static">
+      <Container
+        sx = {{
+          ml: 0,
+        }}
+      >
+        <Toolbar>
+          <Typography
+            variant="h6"
+            noWrap
+            component="a"
+            href="/"
```

Comment:
So I don't know how much you've learned about the VirtualDOM concept, but one small issue here is that since you're using just an href, you're triggering a full page refresh on follow, rather than just changing the VirtualDOM.

You want to leave the href there, so that it can link correctly if someone middle mouse clicks it or command clicks it, but you also want if they click it directly to not trigger a full page refresh if that makes sense. 

The main way to do this is with the `event.preventDefault()` command.

Another thing to be VERY careful with when you use React-Router is to make sure that your history tree remains correct. This is done differently depending on what version of React-Router you use, but the general concept is, you have a function that gets called when you click a link, it does the preventDefault stuff I mentioned earlier, it pushes the new page to the history, then refreshes the VirtualDOM with the new information...

It's a little abstract so if you want help with it, please let me know! I'm happy to help!